macros/contexts/contextBoolean.pl

```diff
@@ -0,0 +1,160 @@
+## contextBoolean.pl
+
+sub _contextBoolean_init { context::Boolean::Init() }
+
+package context::Boolean;
+
+sub Init {
+	my $context = $main::context{Boolean} = Parser::Context->getCopy("Numeric");
+	$context->{name} = "Boolean";
+	$context->{value}{Formula} = "context::Boolean::Formula";
```

Comment:
This should actually be `{parser}{Formula}` if you want the result of `Compute("...")` and student answers to produce a `context::Boolean::Formula`.

Reply:
I just realized there is an issue with doing this.  I need to check some things out and will get back to you.

Reply:
I have noted this change -- what is the difference in behavior between the `context->{value}{Formula}` and `context->{parser}{Formula}` here? 

I encountered this when attempting to accept student answers with a Boolean::Formula object, and now I have both set to my new subclass. So I guess my effective question is "is that necessary?"